Lib/glyphsLib/classes.py

```diff
@@ -1253,8 +1270,16 @@ def name(self):
             if abs(self.italicAngle) > 0.01:
                 names.append("Italic")
             name = " ".join(list(names))
+        self._name = name
         return name
 
+    @name.setter
+    def name(self, value):
+        if self.font and int(self.font.appVersion) >= 1075:
+            self._name = value
+        else:
+            return
```

Comment:
same here. If I set a value, I do set that value, regardless of the input file's appVersion. 
Basically, the appVersion of the input file should only be relevant while parsing, and kept in the file for info only; but it should not (I believe) determine the internal structure of the data nor the output format.